assets/styles.css

```diff
@@ -0,0 +1,177 @@
+body {
+  overflow-x: hidden;
+}
+
+.body-wrapper {
+  width: 1210px;
+  margin: 0 auto;
+}
+
+.header-wrapper {
+  height: 100px;
+  color: white;
+  background-color: #f72210;
+}
+
+.font-head a {
+  font-size: 14px;
+}
+
+.logo-head {
+  color: #fff;
+  fill: white;
+  width: 400px;
+}
+
+.header-font {
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.rounded-pill {
+  background-color: #ffe400;
+  height: 30px;
+}
+
+.search-input {
+  height: 38px;
+  width: max-content;
+}
+
+.article-body {
+  padding-top: 10px;
+  margin-bottom: 15px;
+}
+
+.main-section {
+  padding: 25px;
+}
+
+.navbar-one a {
+  padding: 15px;
+}
+
+.navbar-one li {
+  border-right: 1px solid #dee2e6 !important;
+}
+
+.newsletter-signup {
+  max-width: 300px;
+  margin: 0 auto 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 20px;
+}
+
+.icon-signup {
+  flex: none;
+  display: block;
+  width: 29px;
+  height: 28px;
+  background:
+    url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain
+     no-repeat #f72210;
+  margin-right: 10px;
+}
+
+.free-signup-text {
+  font: bold 19px/19px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+}
+
+.login-text-icon {
+  display: block;
+}
+
+.login-text-logo {
+  display: none;
+}
+
+@media screen and (max-width: 1024px) {
+  .body-wrapper {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+
+@media screen and (max-width: 992px) {
+  .header-wrapper {
+    height: 50px;
+    color: white;
+    background-color: #f72210;
+    max-width: 1310px;
+    padding: 10px;
+  }
+
+  .logo-head {
+    width: 150px;
+    padding-top: 0 !important;
+  }
+}
+
+/* FOOTER CSS */
+
+@media screen and (max-width: 768px) {
+  .footer-links > li {
+    border-right: none !important;
+    padding-top: 0.75rem !important;
+  }
+
+  .footer-copyright {
+    padding-top: 1rem !important;
+  }
+
+  .footer-social {
+    font-size: 20px;
+    padding-top: 0.75rem !important;
+  }
+
+  .login-text-icon {
+    display: none;
+  }
+
+  .login-text-logo {
+    display: block;
+  }
+}
+
+.footer-links {
+  font-size: 0.85rem;
+  line-height: 14px;
+  padding-right: 1rem !important;
+}
+
+.footer-section {
+  border-top: 1px solid #666;
+}
+
+.logo-footer {
+  color: #fff;
+  fill: #f72210;
+  width: 236px;
+}
+
+.footer-copyright {
+  font-size: 0.8rem;
+  line-height: 12px;
+  text-transform: uppercase;
+  color: #666;
+  float: right;
```

Comment:
- [ ] Use `float-right` Bootstrap class instead of `float: right;`

Reply:
done